app/assets/javascripts/discourse/app/lib/is-element-in-viewport.js

```diff
@@ -1,4 +1,7 @@
 export default function (element) {
+  if (!element) {
+    return;
+  }
```

Comment:
unrelated: I wonder if we should clean up usages of this API so we never call it with jQuery objects, seems like a superfluous test, just send it the right type of object.

Reply:
Agreed, removing this